styles/styles.css

```
@@ -56,21 +73,49 @@ body {
     text-transform: uppercase;
 }
 
+.page-conteiner{
+    width: 1160px;
+    margin: 0 auto;
+}
```

```suggestion
.page-container {
```
Так будет правильно по-английски.